src/App.js

```diff
@@ -18,16 +19,19 @@ const Wrapper = styled.div`
 
 function App() {
   return (
-    <ThemeProvider theme={mainTheme}>
-      <BrowserRouter>
-        <Switch>
-          <Wrapper>
-            <Route exact path="/" component={Home} />
-          </Wrapper>
-          <Route exact path="/home" component={Home} />
-        </Switch>
-      </BrowserRouter>
-    </ThemeProvider>
+    <Provider>
+      <ThemeProvider theme={mainTheme}>
+        <BrowserRouter>
+          <Switch>
+            <Wrapper>
```

Comment:
O que seria esse Wrapper?

```suggestion
            <Wrapper>
```